yapapi/services.py

```diff
@@ -779,6 +821,10 @@ def spawn_instances(
             In other words, if both `num_instances` and `instance_params` are provided,
             the number of instances spawned will be equal to `num_instances` and if there are
             too few elements in the `instance_params` iterable, it will results in an error.
+        :param network_addresses: optional list of network addresses in case the Cluster is
+            attached to VPN. If the list is not provided (or if the number of elements is less
+            than the number of spawned instances), any instances for which the addresses have not
+            been given, will be assigned an address automatically.
```

Comment:
okay, I don't think it makes sense to resolve this issue in this pull request ... plus, there _is_ a reason why it must be part of "spawn_instances" -> because `spawn_instances` can be called on the `Cluster` itself too

Reply:
I don't think this is a good reason. Why `Cluster.spawn_instances` would need to have exactly the same interface as `Golem.run_service`? They seem to be totally different things, despite first being called by the second.

But I agree this is not a part of this PR.

Reply:
they don't _need_ to have exactly the same interface ... but if they (partially) do the same thing, the interface for those pieces should stay consistent